README.md

```diff
@@ -320,13 +320,16 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
   ilab model download
   ```
 
-  `ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace and store it in a `models` directory:
+  `ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace:
 
   ```shell
   (venv) $ ilab model download
-  Downloading model from instructlab/merlinite-7b-lab-GGUF@main to models...
-  (venv) $ ls models
-  merlinite-7b-lab-Q4_K_M.gguf
+  Downloading model from Hugging Face: instructlab/merlinite-7b-lab-GGUF@main to /home/user/.cache/instructlab/models...
+  Downloading 'merlinite-7b-lab-Q4_K_M.gguf' to '/home/user/.cache/instructlab/models/.huggingface/download/merlinite-7b-lab-Q4_K_M.gguf.9ca044d727db34750e1aeb04e3b18c3cf4a8c064a9ac96cf00448c506631d16c.incomplete'
+  INFO 2024-08-01 15:03:22,852 huggingface_hub.file_download:1877: Downloading 'merlinite-7b-lab-Q4_K_M.gguf' to '/home/user/.cache/instructlab/models/.huggingface/download/merlinite-7b-lab-Q4_K_M.gguf.9ca044d727db34750e1aeb04e3b18c3cf4a8c064a9ac96cf00448c506631d16c.incomplete'
+  merlinite-7b-lab-Q4_K_M.gguf: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4.37G/4.37G [02:25<00:00, 30.0MB/s]
+  Download complete. Moving file to /home/user/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf
+  INFO 2024-08-01 15:05:48,464 huggingface_hub.file_download:1893: Download complete. Moving file to /home/user/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf
```

Comment:
@danmcp this is the expected output of `ilab model download` for a merlinite .gguf. Not sure if there is anything besides the name of the user we'd want to change or obfuscate in this section of the README.

Reply:
I wasn't trying to say you needed to include all the output.  I am not sure I prefer that given it looks like a bunch of duplicated messages (could use some work).  It just wasn't clear why the ... was being added.  Do you have a preference?

Reply:
The `...` is used throughout the README to signify that there is more output besides what is being shown in the section. I changed this section to include the first and last line to show more complete output.